clippy_dev/src/bless.rs

```diff
@@ -69,23 +49,14 @@ fn update_reference_file(reference_file_path: PathBuf, test_name: PathBuf, ignor
 
     if test_output_file != reference_file {
         // If a test run caused an output file to change, update the reference file
-        println!("updating {}", &relative_reference_file_path.display());
+        println!("updating {}", reference_file_path.display());
         fs::copy(test_output_path, &reference_file_path).expect("Could not update reference file");
-
-        // We need to re-read the file now because it was potentially updated from copying
-        let reference_file = fs::read(&reference_file_path).unwrap_or_default();
-
-        if reference_file.is_empty() {
-            // If we copied over an empty output file, we remove the now empty reference file
-            println!("removing {}", &relative_reference_file_path.display());
-            fs::remove_file(reference_file_path).expect("Could not remove reference file");
-        }
```

Comment:
This looks good, but do you mind explaining why this code can be removed to make sure I understand?

Reply:
Can do!

I'm imagining an older version of compiletest could create empty `.stderr/stdout` files in `target`, in which case this code would prevent leaving an empty file in the test directories

I was testing that I hadn't broken this part by commenting out code for a `.stderr` and a `#[clippy::author]` for `.stdout`. I found that compiletest would remove the file rather than produce an empty one, so there would be no copied empty reference file to remove